README.md

```diff
@@ -506,6 +506,22 @@ try, or use as a reference for your own implementation. One of them,
 enables you to use a different browser for authentication, like Chrome for iOS
 or Firefox for iOS.
 
+##### CocoaPods
+Include the `EnterpriseUserAgent` subspec alongside any pods/subspecs you were already using, e.g.:
+```
+	pod 'AppAuth'
+	pod 'AppAuth/EnterpriseUserAgent'
+```
+Make sure to import `AppAuthEnterpriseUserAgent` in addition to `AppAuth` if you are using the full `AppAuth` functionality.
+
+##### Carthage
+Use the `EnterpriseUserAgent` Framework, which includes all the headers of the `AppAuth` framework.
```

Comment:
I think we should call the framework 'AppAuthEnterpriseUserAgent' to match CocoaPods documented above, and SPM documented below.